spec/tweetstream/client_authentication_spec.rb

```diff
@@ -2,17 +2,19 @@
 
 describe TweetStream::Client do
   before do
-    @stream = double('EM::Twitter::Client',
-                     :connect => true,
-                     :unbind => true,
-                     :each => true,
-                     :on_error => true,
-                     :on_max_reconnects => true,
-                     :on_reconnect => true,
-                     :connection_completed => true,
-                     :on_no_data_received => true,
-                     :on_unauthorized => true,
-                     :on_enhance_your_calm => true,
+    @stream = double(
+      'EM::Twitter::Client',
+      :connect => true,
+      :unbind => true,
+      :each => true,
+      :on_error => true,
+      :on_max_reconnects => true,
+      :on_reconnect => true,
+      :connection_completed => true,
+      :on_no_data_received => true,
+      :on_unauthorized => true,
+      :on_enhance_your_calm => true,
+      :on_close => true,
```

Comment:
there's a few diffs with changes like this, this just adds diff-noise to the commit, if possible, can you please revert this and the other instances of indentation changes?